integration-test/project.json

```diff
@@ -0,0 +1,170 @@
+{
+	"info": {
+		"_postman_id": "ef08e91f-4c54-4535-8cb2-c67eafa3008c",
+		"name": "project-integration-test",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "23207395",
+		"_collection_link": "https://planetary-sunset-570862.postman.co/workspace/DevLaunchers~ccbd0f9c-ee35-4275-b2ce-f2bfbb1fd054/collection/23207395-ef08e91f-4c54-4535-8cb2-c67eafa3008c?action=share&source=collection_link&creator=23207395"
+	},
+	"item": [
+		{
+			"name": "login",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"const response = pm.response.json();",
+							"",
+							"",
+							"pm.test(\"Response time is less than 200ms\", function () {",
+							"    pm.expect(pm.response.responseTime).to.be.below(200);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n  \"identifier\": \"integration-test@devlaunchers.org\",\n  \"password\": \"integration-test\"\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{base_url}}/auth/local",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"auth",
+						"local"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "get projects",
```

Comment:
Nit: I would call this list projects, because the endpoint return multiple projects. `get project` would be something like `/projects/1`